openrest4j-api/src/main/java/com/openrest/olo/dispatches/PickupDispatchInfo.java

```diff
@@ -13,11 +15,16 @@
     private static final long serialVersionUID = 1L;
 
     /** Default constructor for JSON deserialization. */
-    public PickupDispatchInfo() {}
+    public PickupDispatchInfo() {
+        this.withCurbsideInfo = false;
+    }
 
     public PickupDispatchInfo(Integer minOrderPrice, Integer charge, Integer delayMins, Boolean inactive,
-                              Availability availability, Map<String, String> properties) {
+                              Availability availability, Map<String, String> properties, CurbsideInfo curbsideInfo,
+                              Boolean withCurbsideInfo) {
         super(minOrderPrice, charge, delayMins, inactive, availability, properties);
+        this.withCurbsideInfo = (withCurbSideInfo != null) ? withCurbSideInfo : false;

```

Comment:
simple assign is enough - no need for != null

Reply:
done